src/start/input_cp2k_motion.F

```diff
@@ -887,7 +898,7 @@ RECURSIVE SUBROUTINE create_geoopt_section(section, location, label, description
       ! collects keywords related to reduction of symmetry due to an external field
       CALL keyword_create( &
          keyword, __LOCATION__, name="SYMM_REDUCTION", &
-         description="Polarisation vector of electric field for symmetry reduction", &
+         description="Polarization vector of the electric field for space group symmetry reduction", &
```

Comment:
@ltalirz This is indeed intended for calculations with an applied electric field. The former partially breaks the symmetry and this needs to be taken into account. It is not very elegant but I did not know how to easily access this information in FIST and QS. Furthermore, the electric field is defined differently in FIST and QS. In one, it is a fractional (scaled) vector, and in the other it is given directly in cartesian coordinates. Finally, having this option in SPGR, let users break/lower the symmetry if they want/need to.

Reply:
@ltalirz Further clarifications here. The space group that is identified is the same with or without the electric field. However, the former forbids some symmetry operations of that space group. This keyword is used to determine which of the space group symmetry operations are forbidden to prevent them from being enforced.